include/oneapi/dpl/pstl/omp/parallel_merge.h

```diff
@@ -13,8 +13,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _ONEDPL_INTERNAL_OMP_PARALLEL_MERGE_H
-#define _ONEDPL_INTERNAL_OMP_PARALLEL_MERGE_H
+#ifndef _ONEDPL_PARALLEL_MERGE_H
```

Comment:
I think it is potentially dangerous. "OMP" substring is omitted.
What else if "tomorrow" we will add another host backend (another folder) with similar filenames and guards?

Reply:
I propose to think about this situation tomorrow, if we really will have this case.
And now we may just follow to our existing guard names rule.

Reply:
UPD: `_OMP` restored.